pkg/sysctl/sysctl.go

```diff
@@ -54,7 +60,28 @@ func parameterPath(name string) (string, error) {
 			return "", ErrInvalidSysctlParameter(name)
 		}
 	}
-	return filepath.Join(append([]string{prefixDir}, elems...)...), nil
+	return filepath.Join(append([]string{GetProcfs(), "sys"}, elems...)...), nil
+}
+
+// SetProcfs sets path for the root's /proc. Setting this value with a different
+// default value and executing it after GetProcfs causes panic.
```

Comment:
This comment is still referring to the previous behavior
```suggestion
// SetProcfs sets path for the root's /proc. Calling it after GetProcfs causes panic.
```